src/Raven.Server/Utils/PoolOfThreads.cs

```diff
@@ -270,21 +270,21 @@ private bool ResetThreadAffinity()
 
             private static bool ResetThreadPriority()
             {
-                try
-                {
-                    if (Thread.CurrentThread.Priority != ThreadPriority.Normal)
-                        Thread.CurrentThread.Priority = ThreadPriority.Normal;
-                    return true;
-                }
-                catch (Exception e)
+                if (Thread.CurrentThread.Priority != ThreadPriority.Normal)
```

Comment:
Do we need to worry about _reading_ the `Priority` failing?

Reply:
done